opm/simulators/wells/RateConverter.hpp

```diff
@@ -451,6 +451,17 @@ namespace Opm {
 
                 }
 
+                // quantities for pore volume average
+                std::unordered_map<RegionId, Attributes> atrributes_pv;
```

Comment:
I think this version is much better, and avoids the many unnecessary lookups of the previous iteration. Good!